salesagility/SuiteCRM
# Fix PipelineBySalesStage not converting currency

The PipelineBySalesStageDashlet was not converting currency when determining amounts shown in the chart. However, it was changing the currency symbol to the user's selected currency. This is a bug as the user's currency could be the Japanese Yen, so it would show the Yen symbol (¥) next to US Dollar amounts.

## Description
Just adjusting the query that grabs the values for the chart so that the query converts the us_dollar amounts to the user's default currency. If user has no default currency, we use the system's default currency.

### Steps to reproduce bug
Prereqs: Have user default currency in USD
1. Add the PipelineBySalesStageDashlet to dashboard
2. Notice currency symbol and amounts
3. Change default currency to something else
4. Go back to dashboard
5. Notice that the currency symbol has changed BUT the value amounts are still in US dollars

## Motivation and Context
To show the correct currency conversion with the correct currency symbol in the PipelineBySalesStageDashlet.

## How To Test This
Prereqs: Have user default currency in USD
1. Add the PipelineBySalesStageDashlet to dashboard
2. Notice currency symbol and amounts
3. Change default currency to something else
4. Go back to dashboard
5. Notice that the currency symbol and amounts have appropriately changed

## Types of changes


### Final checklist
